.buildkite/deploy.yml

```diff
@@ -2,12 +2,22 @@ steps:
   - label: ":package: Build image"
     key: build
     command:
-      - "docker build -t ${CONTAINER_NAME} -f ./${CONTAINER_NAME}/Dockerfile ."
+      - "docker build -t ${CONTAINER_NAME}:{{matrix}}
```

Comment:
Do we need `\` at the end of each line to escape the new line? (this applies in other parts of this PR too.

Reply:
It reads multi-line commands without needing `\` as long as its specified in the same index `-`